src/App.js

```diff
@@ -10,10 +11,35 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedTab: tabs[0],
+  }
+
+  onTabSelected = (tabId) => {
+    const newTab = tabs.find(tab => tab.id === tabId);
+
```

Comment:
As task description says:

> Don't call the callback if the tab was not changed
So you should add the check if clicked tab is === seledted tab